clar2wasm/src/standard/standard.wat

```diff
@@ -661,4 +798,16 @@
     (export "ge-int" (func $ge-int))
     (export "log2-uint" (func $log2-uint))
     (export "log2-int" (func $log2-int))
+    (export "bit-and-uint" (func $bit-and))
+    (export "bit-and-int" (func $bit-and))
+    (export "bit-not-uint" (func $bit-not))
+    (export "bit-not-int" (func $bit-not))
+    (export "bit-or-uint" (func $bit-or))
+    (export "bit-or-int" (func $bit-or))
+    (export "bit-xor-uint" (func $bit-xor))
+    (export "bit-xor-int" (func $bit-xor))
+    (export "bit-shift-left-uint" (func $bit-shift-left))
+    (export "bit-shift-left-int" (func $bit-shift-left))
```

Comment:
Do we need both in this case?

Reply:
Lots of the surrounding code depends on both being available. The overhead of two exports is negligable

Reply:
There are places in the codebase that just assume every function has a int and uint version